VideoWeb/VideoWeb/ClientApp/src/app/testing/mocks/mock-consultation-service.ts

```diff
@@ -22,5 +23,7 @@ export function consultationServiceSpyFactory(): jasmine.SpyObj<ConsultationServ
     consultationServiceMock.leaveConsultation.and.returnValue(Promise.resolve());
     consultationServiceMock.respondToAdminConsultationRequest.and.returnValue(Promise.resolve());
     consultationServiceMock.startPrivateConsulationWithEndpoint.and.returnValue(Promise.resolve());
+    consultationServiceMock.consultationAcceptedBy = new BehaviorSubject<boolean>(true);
```

Comment:
Again this can be removed since the getConsultationMessage already does this job

Reply:
removed